README.md

```diff
@@ -1 +1,45 @@
-# Hello-world
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Hello-World
+
+> This project is a simple Html and Css project.
+
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple steps:
+1. Visit my profile https://github.com/gauravgangwarweb/Hello-world
+2. On your terminal/IDE use the command
+   "git clone https://github.com/mckent05/Hello_Project.git"
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@githubhandle](https://github.com/gauravgangwarweb)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
[OPTIONAL] Kindly check the link for the project MIT is not working. Please fix it :wrench: so the readme file is simple, readable, and professional. :smiley_cat: